action.yml

```diff
@@ -80,12 +80,20 @@ inputs:
       Please refer to this action's README for all available placeholders.
     default: >-
       [Backport ${target_branch}] ${pull_title}
+  pull_branchname:
+    description: ""
+    default: >-
+      backport-${pull_number}-to-${target}
```

Comment:
❌ I think this change doesn't fully achieve what you intend.

This default was a template literal (or template string), where the variables `${pull_number}` and `${target}` are interpolated on execution. 

```ts
const branchname = `backport-${pull_number}-to-${target}`;
```

But now the `pull_branchname` is a special input where placeholders can be replaced with values. These are defined by utils:

https://github.com/korthout/backport-action/blob/930286d359d53effaf69607223933cbbb02460eb/src/utils.ts#L3-L21

Note that `${pull_number}` is a valid placeholder, but `${target}` is not. 

💡 Instead of `${target}`, you need to use `${target_branch}`.

Reply:
Nice catch! 👍